aspnetcore/fundamentals/routing/samples/3.x/RoutingSample/EndpointInspectorStartup.cs

```diff
@@ -13,14 +13,14 @@ namespace RoutingSample
 {
     public class EndpointInspectorStartup
     {
+        #region snippet
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
             }
 
-            #region snippet
             app.UseRouting();
 
             app.Use(next => context =>
```

Comment:
Sorry, I've only just seen this. It looks like I was no longer subscribed to this issue...

For this sample, I'd just change it to be non-terminal, i.e. `await next(context)` so that the pipeline progresses. I can submit that as a follow-up PR, if that's something that makes sense. Feel free to apply it to this PR if what I'm trying to say is clear.

Reply:
@rynowak  please review @serpent5  suggestion.

Reply:
👍 from me